includes/utilities.inc

```diff
@@ -44,6 +47,7 @@ function islandora_usage_stats_log_object_view(AbstractObject $object) {
         try {
           $collection_id = islandora_usage_stats_get_pid_id($collection);
           islandora_usage_stats_collection_view_to_db($object_access_id, $collection_id);
+          module_invoke_all('islandora_usage_stats_access_recorded', 'collection', $collection_id);
```

Comment:
I think this part can be a bit confusing, since it sounds like you are recording when collections are accessed, but what you are truly recording is the parent of the object being accessed. For this reason, I'd change the "$type" param from "collection" to "parent" to make it more explicit what is being recorded here.

Reply:
I see your point, but my intent was to make the values of `$type` consistent with the rest of the code, which uses 'collection'. I'll double check, but my understanding is that when an object is accessed, the hit is counted against the collection only, and not against anything else that happens to be the object's parent structurally; in other words, a hit is not recorded against a compound object when one if its child objects is accessed. Makes no difference to me which value we use though, as long as it's clear.

Maybe "parent_collection" is the best of both?

Reply:
@mjordan That sounds like a great idea to me, clearer names are always better.

Reply:
Great, I'll use that.